### PR TITLE
feat(skills): add skill section on home

### PR DESCRIPTION
**Pull request by mfrachet**
screenshot 2019-01-17 at 09 24 32

### CODE REVIEW

File: i18n/fr.json [hunk 1]
```diff
@@ -81,5 +83,8 @@
     "title": "Révolution du numérique",
     "subtitle": "Co-construire",
     "content": "L’innovation technologique, organisationnelle et managériale permettant de construire ensemble un monde plus simple, plus ouvert, plus libre et plus accessible à tous."
-  }
+  },
+  "advice": "Conseil",
```
Comment (l-bonanni) [inline i18n/fr.json]: shouldn't we add them in the skills part ?

File: i18n/fr.json [hunk 2]
```diff
@@ -40,6 +40,8 @@
     }
   },
   "skills": {
+    "title": "Triple compétence",
```
Comment (l-bonanni) [inline i18n/fr.json]: ⚠️ don't forget the english part :)

File: components/modules/Skills.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <app-section>
+      <app-cols>
+        <app-col :size="1" v-for="skill in skills" :key="skill.label">
+          <app-figure :title="$t(skill.label)" :image="skill.image" size="33%"></app-figure>
+        </app-col>
+      </app-cols>
+    </app-section>
+    <app-hero size="small">
```
Comment (l-bonanni) [inline components/modules/Skills.vue]: 👍

File: components/Figure.vue [hunk 4]
```diff
@@ -1,13 +1,38 @@
...
+  font-size: 2em;
+  margin-top: -2em;
+}
+
+figure {
+  display: block;
+  margin: 0;
+}
+
+.figure-image {
```
Comment (l-bonanni) [inline components/Figure.vue]: no need of class here, just write `figure img {`

File: components/Figure.vue [hunk 5]
```diff
@@ -1,13 +1,38 @@
...
+  }
 };
 </script>
 
+<style scoped>
+figcaption {
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 2em;
+  margin-top: -2em;
```
Comment (l-bonanni) [inline components/Figure.vue]: 🤢

File: components/Figure.vue [hunk 6]
```diff
@@ -1,13 +1,38 @@
...
+}
+
+figure {
+  display: block;
+  margin: 0;
+}
+
+.figure-image {
+  object-fit: cover;
```
Comment (l-bonanni) [inline components/Figure.vue]: 😍

File: components/Figure.vue [hunk 7]
```diff
@@ -1,13 +1,38 @@
...
+  width: 100%;
+  height: 50vh;
```
Comment (l-bonanni) [inline components/Figure.vue]: it will not necessarily depend on the viewport height... WYT ?

File: i18n/fr.json (hunk 1 above)
Comment (mfrachet) [reply i18n/fr.json]: `advice` can be used anywhere, but not only the section. WYT?

File: i18n/fr.json (hunk 2 above)
Comment (mfrachet) [reply i18n/fr.json]: Yes, but I think that we should write the full FR part, and then C/C. It's such a pain to write on each part :p

File: components/Figure.vue (hunk 5 above)
Comment (mfrachet) [reply components/Figure.vue]: How should I make it ?

File: i18n/fr.json (hunk 1 above)
Comment (l-bonanni) [reply i18n/fr.json]: Alright

File: components/Figure.vue (hunk 5 above)
Comment (l-bonanni) [reply components/Figure.vue]: I suggest you that : figure with a position relative and figcaption with a position absolute, left and right position to 0 and a line-height. This way, with a props you can place it everywhere you want. (For example, a props bottom will add the style "bottom: 0"

**Pull request merged**